uber/model_checks.py

```diff
@@ -23,6 +23,8 @@ def duplicate_admin(account):
         with Session() as session:
             if session.query(AdminAccount).filter_by(attendee_id=account.attendee_id).all():
                 return 'That attendee already has an admin account'
+            elif session.query(Attendee).filter_by(id=account.attendee_id).first().email == '':
+                return "Attendee doesn't have a valid email set"
 
```

Comment:
I think it might be a good idea to put this in a new validator that is named 'check_valid_email_exists', or something like that. (since this one is named duplicate_admin, and this check isn't related to that)